db/schema.rb

```diff
@@ -246,6 +249,7 @@
     t.string "hashed_sign_in_token"
     t.boolean "consent_to_be_contacted"
     t.string "feedback_status", default: "not_requested_yet", null: false
+    t.jsonb "questionnaire"
```

Comment:
You got some unrelated stuff in here I think.

Reply:
Thanks!